File: packages/sdk/sdk-py/examples/chat/stream.py
import os
from phaseo import Phaseo

api_key = os.getenv("PHASEO_API_KEY")
if not api_key:
	raise RuntimeError("Set PHASEO_API_KEY")

client = Phaseo(api_key=api_key)

stream = client.chat.completions.create({
	"model": "openai/gpt-5-nano",
	"messages": [{"role": "user", "content": "Write a short greeting."}],
	"stream": True,
})

for line in stream:
	if line == "data: [DONE]":
		break
	print(line)

File: packages/sdk/sdk-ts/examples/chat/stream.ts
import { Phaseo } from "../../src";

const apiKey = process.env.PHASEO_API_KEY;
if (!apiKey) throw new Error("Set PHASEO_API_KEY");

async function main() {
	const client = new Phaseo({ apiKey });
	const stream = await client.chat.completions.create({
		model: "openai/gpt-5-nano",
		messages: [{ role: "user", content: "Write a short greeting." }],
		stream: true,
	});

	for await (const line of stream as AsyncGenerator<string>) {
		if (line === "data: [DONE]") break;
		console.log(line);
	}
}

main().catch((err) => {
	console.error(err);
	process.exitCode = 1;
});
